### ISSUE TITLE
home-manager fails on Darwin with nixpkgs-unstable

### ISSUE THREAD

**Issue by danieldk** [rycee/home-manager#324]
Running `home-manager switch` on Darwin with nixpkgs-unstable fails:

~~~
$ home-manager switch
error: assertion failed at /Users/daniel/git/nixpkgs/pkgs/top-level/stage.nix:153:21
(use '--show-trace' to show detailed location information)
~~~

This seems to be caused by `pkgsi686Linux` now asserting that the host platform is Linux:

https://github.com/NixOS/nixpkgs/blob/920ad8b3d8be6ce9a350a7b735ee7a70f1b36ad3/pkgs/top-level/stage.nix#L153

I am new to nix and home-manager, so I don't know what a proper fix is, but commenting out the following line:

~~~
pkgs_i686 = _pkgs.pkgsi686Linux;
~~~

in `modules/misc/nixpkgs.nix` solves the problem.

**Comment by rycee** [comment]
Perhaps you are using an old checkout of Home Manager? This issues should be fixed as of 092706eff85aa4501994016375693320ef9d2e7b.

**Comment by danieldk** [comment]
Ah, I started with home-manager from nixpkgs I think, which seems to be from June 14. Everything works fine now, sorry for the noise!
